sfge/src/main/java/com/salesforce/graph/ops/GraphUtil.java

```diff
@@ -181,10 +182,9 @@ private static List<Util.CompilationDescriptor> buildFolderComps(String sourceFo
                     "Could not read file/directory " + sourceFileVisitor.lastVisitedFile, ex);
         } catch (JorjeUtil.JorjeCompilationException ex) {
             throw new GraphLoadException(
-                    "Could not parse file "
-                            + sourceFileVisitor.lastVisitedFile
-                            + ": "
-                            + ex.getMessage(),
+                    String.format(
+                            UserFacingMessages.FIX_COMPILATION_ERRORS,
```

Comment:
How do we handle cases where multiple files have syntax errors? Do we display only one file's errors at a time?

Reply:
Since we parse files one-by-one, we'll only show the syntax errors for the first file that has any.